src/components/DataItem/DataItem.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import Paper from '@mui/material/Paper';
+import Box from '@mui/material/Box';
+import { cx, css } from '@emotion/css';
+import { tokens } from '../../theme/tokens';
+
+import { Typography } from '../Typography';
+
+export type DataItemsProps = {
+  icon: string;
+  title: string;
+  description?: string;
+  className?: string;
+}
+
+export const DataItem = React.forwardRef<HTMLDivElement, DataItemsProps>(function DataItem(
+  {icon, title, description, className, ...props},
+ref,
+  ) {
+  return(
+    <Paper
+      variant="outlined"
+      sx={{ display: 'flex' }}
+      className={cx('mb-5', className)}
+      ref={ref}
+    >
+      <img src={icon} alt="" className={styles.icon(tokens.colors.shade)} />
+      <Box sx={{ flex: '1 0 auto', p: 2, pb: 2.25 }}>
+        <div>
```

Comment:
Why do we need another div here?

Reply:
some despair testing was there, nice catch :D